utils/update_build_version.py

```diff
@@ -30,13 +30,32 @@
 
 OUTFILE = 'build-version.inc'
 
+
+def command_output(cmd, dir):
+    """Runs a command in a directory and returns its standard output stream.
+
+    Raises a RuntimeError if the command fails to launch or otherwise fails.
+    """
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, cwd=dir)
```

Comment:
To avoid spamming stderr if git-describe fails, you could stderr=subprocess.PIPE as well.